examples/node-app-mssql/README.md

```diff
@@ -10,12 +10,79 @@ Install dependencies.
 $ yarn
 ```
 
-Configure database connection(s) in the `connections.sync-db.json`. 
+Configure database connection(s) in the `connections.sync-db.json`.
 
 ```bash
 $ cp connections.sync-db.json.example connections.sync-db.json
 ```
 
+## Migrations
+
+Create a configuration file `knexfile.ts` in the root directory.
+
+```js
+import { connections } from "./connections.sync-db";
```

Comment:
We prefer `single-quotes`. Let's use single quotes in the js code snippets README as well?